src/button-icons/button-icons.css

```diff
@@ -27,15 +27,13 @@
 .omrs-btn-icon-large svg {
   position: relative;
   right: 0.5rem;
-  fill: var(--omrs-color-primary-blue);
```

Comment:
Button icons should assume the default icon color instead of forcing blue. This might break somewhere in esm-login or esm-home that is assuming it's blue, but it's worth it imo because we need it to be easy to change the color of button icons.